toolset/utils/metadata.py

```diff
@@ -12,7 +12,7 @@ class Metadata:
 
     supported_dbs = []
     for name in databases:
-        supported_dbs.append((name, databases[name].description))
+        supported_dbs.append((name, '...'))
```

Comment:
Is the second part of this tuple ever read? It should probably be removed and anything that relies on it made aware of that change.

Reply:
Yes. The `allowed` property for all items in the benchmark config are expected to be tuples, not just databases. Makes it easier to check each benchmark config property with the same code: https://github.com/TechEmpower/FrameworkBenchmarks/blob/master/toolset/utils/metadata.py#L372

A couple of other properties are built the same way: https://github.com/TechEmpower/FrameworkBenchmarks/blob/master/toolset/utils/metadata.py#L295